sci-mathematics/yafu/yafu-1.34.3-r2.ebuild

```
@@ -0,0 +1,54 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic edos2unix toolchain-funcs
+
+MY_PV="$(ver_cut 1-2)"
+DESCRIPTION="Yet another factoring utility"
+HOMEPAGE="https://sourceforge.net/projects/yafu/"
+SRC_URI="https://downloads.sourceforge.net/${PN}/${MY_PV}/${PN}-${MY_PV}-src.zip"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="
+	|| (
+		>=sys-devel/gcc-4.2:*[openmp]
+		llvm-core/clang-runtime:*[openmp]
+	)
+	dev-libs/gmp:0=
+	sci-mathematics/gmp-ecm"
+RDEPEND="${DEPEND}"
+BDEPEND="app-arch/unzip"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-Makefile.patch
+	"${FILESDIR}"/${P}-unshadow-toolchain.patch
+	"${FILESDIR}"/${P}-seds.patch
+	"${FILESDIR}"/${P}-mpz_set.patch
+	)
+
+src_prepare() {
+	edos2unix factor/factor_common.c top/eratosthenes/soe_util.c
+	default
+}
+
+src_configure() {
+	tc-export CC
+	append-cflags -fcommon
+	default
+}
+
+src_compile() {
+	local VAR
+	use amd64 && emake $VAR x86_64
```

```suggestion
	use amd64 && emake ${VAR} x86_64
```
... but `${VAR}` isn't used for anything? What is this for?

---

I think this is for me not looking over stuff that "works" in ported code.